### DISCUSSION TITLE
Window position and size

### DISCUSSION THREAD

**Discussion by notDavid** [marta-file-manager/marta-issues#888] [Q&A]
Hi there,

Is it possible to set / save / remember the last Window position and size?

If I change the window size to fit my screen, it seems to not be remembered. Anytime i close and reopen a new Window, or close and open the app, the window is positioned back to the "stock default".

I enjoy the app, thank you :-)

**Comment by sleepymalc** [comment]
Can you be more explicit? I don't have this issue after quitting and re-opening the app. A quick fix would be using `cmd+h` to hide the window rather than `cmd+q` for quitting.

**Comment by airfortech** [comment]
same for built terminal. Every restart toggle terminal shows it almost fullscreen. Nice to have option save size of terminal.
